mlflow/paddle/__init__.py

```diff
@@ -254,12 +265,30 @@ def load_model(model_uri):
     local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
     flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
     pd_model_artifacts_path = os.path.join(local_model_path, flavor_conf["pickled_model"])
-    return paddle.jit.load(pd_model_artifacts_path)
+    if not model:
+        return paddle.jit.load(pd_model_artifacts_path, **kwargs)
+    elif not isinstance(model, paddle.Model):
+        raise TypeError(
+            "Invalid object type `{}` for `model`, must be `paddle.Model`".format(type(model))
+        )
+    else:
+        support_retrain = _check_if_model_supports_retrain(local_model_path)
+        if not support_retrain:
+            raise TypeError(
+                "The model doee not support re-train. \
+                Please set flag training=True when the paddle model is saved"
+            )
```

Comment:
I'm trying to come up with a nice error message here.

Reply:
Sorry for my pool English...

Reply:
```suggestion
            raise TypeError(
                "This model can't be loaded via `model.load` because a '.pdparams' file doesn't exist. "
                "Please leave `model` unspecified to load the model via `paddle.jit.save` "
                "or set `training` to True when saving a model."
            )
```

Reply:
How about "Saved model doesn't support re-train. Please set flag `training=True` when the `paddle.Model` model is saved i.e., `mlflow.paddle.train(..., training=True)`. Or set `model=None` for inference only."

Reply:
> Sorry for my poor English...

Don't worry, I'm not good at Enlighs at all either and always struggling to write comments and docs.

Reply:
The reason I avoided using `re-train` is we're not sure what the user does after loading the model.

Reply:
I see. Yeah, it can make the users confusing.

Reply:
btw if we try to load a model that doesn't have a `.pdparams` file, what error message do we get?